fastly/block_fastly_service_v1_splunk_test.go

```diff
@@ -572,6 +584,8 @@ resource "fastly_service_v1" "foo" {
     placement          = "waf_debug"
     tls_hostname       = "example.com"
     tls_ca_cert        = %q
+    tls_client_cert    = %q
```

Comment:
@kellymclaughlin does it matter that the CA cert is the same as the Client cert? I guess not, as in these tests we're not validating the server certificate (i.e. its authenticity) using the given CA certificate.

Could we just omit the `tls_ca_cert` property from the TF here to avoid confusion or is there merit in keeping it?

Reply:
We need to keep the CA cert as it's part of the possible. config that is necessary: See also https://docs.fastly.com/en/guides/log-streaming-kafka#adding-kafka-as-a-logging-endpoint

Reply:
Reusing the certificate here is valid behavior in the sense that our validations only look for PEM formatted data. I think the docs make it clear about the different purpose of these two fields, but I see your point. How about I just add a comment here indicating that this is only for testing to avoid anyone looking at the test and copying the behavior?

Reply:
That works for me 👍🏻 Thanks @kellymclaughlin

Reply:
@Integralist I added those comment. If those look good to you I'll kick off a full test suite run to finish this off.